drivers/ninaw10/nina_wifi_bsp.c

```diff
@@ -46,67 +46,82 @@
 #define debug_printf(...)
 #endif
 
-int nina_bsp_init(void) {
-    mp_hal_pin_output(MICROPY_HW_NINA_GPIO1);
+// Use the name CS instead of GPIO1
+#ifndef MICROPY_HW_NINA_CS
+#define MICROPY_HW_NINA_CS MICROPY_HW_NINA_GPIO1
+#endif
```

Comment:
I feel like it's not worth having this alias.  Just leave it as GPIO1.

(I'm trying to get minimal changes to this driver, to make this PR easier to understand, and less chance to break existing users of ninaw10.)

Reply:
OK, makes sense.

Maybe it's worth putting the above table in https://github.com/micropython/micropython-lib/pull/642 ?